supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorWrapper.scala

```diff
@@ -153,4 +153,5 @@ object HtCondorRuntimeKeys {
   val Disk = "request_disk"
   val LogXml = "log_xml"
   val LeaveInQueue = "leave_in_queue"
+  val InitialWorkingDir = "Iwd"
```

Comment:
IMO it's a nice consistency to have runtime keys start lowercase and be full words (i.e. `initial_working_directory` instead of `Iwd`)